site/docs/reference/Connectors/materialization-connectors/amazon-redshift.md

```diff
@@ -0,0 +1,122 @@
+| `/schema`                      | Alternative Schema          | Alternative schema for this table (optional).                                         | string  |                  |
+| `/additional_table_create_sql` | Additional Table Create SQL | Additional SQL statement(s) to be run in the same transaction that creates the table. | string  |                  |
+
+### Sample
+
+```yaml
+materializations:
+  ${PREFIX}/${mat_name}:
+    endpoint:
+      connector:
+        image: "ghcr.io/estuary/materialize-redshift:dev"
+        config:
+          address: "redshift-cluster.account.us-east-2.redshift.amazonaws.com:5439"
+          user: user
+          password: password
+          database: db
+          awsAccessKeyId: access_key_id
+          awsSecretAccessKey: secret_access_key
+          bucket: my-bucket
+          region: us-east-2
+    bindings:
+      - resource:
+          table: ${TABLE_NAME}
+        source: ${PREFIX}/${COLLECTION_NAME}
+```
+
+## Setup
+
+You must configure your cluster to allow connections from Estuary. This can be accomplished by
+making your cluster accessible over the internet for Estuary Flow's IP address `34.121.207.128`, or
+using an SSH tunnel. Connecting to the S3 staging bucket does not use the network tunnel and
+connects over HTTPS only.
+
+Instructions for making a cluster accessible over the internet can be found
+[here](https://aws.amazon.com/premiumsupport/knowledge-center/redshift-cluster-private-public/).
+When using this option, database connections are made over SSL only.
+
+For allowing secure connections via SSH tunneling:
+
+1. Refer to the [guide](../../../../guides/connect-network/) to configure an SSH server on using an
+   AWS EC2 instance.
+
+2. Configure your connector as described in the [configuration](#configuration) section above, with
+the additional of the `networkTunnel` stanza to enable the SSH tunnel, if using. See [Connecting to
+endpoints on secure
+networks](../../../concepts/connectors.md#connecting-to-endpoints-on-secure-networks) for additional
+details and a sample.
+
+## Naming Conventions
+
+Redshift has requirements for [names and
+identifiers](https://docs.aws.amazon.com/redshift/latest/dg/r_names.html) and this connector will
+automatically apply quoting when needed. All table identifiers and column identifiers (corresponding
+to Flow collection fields) are treated as lowercase. Table names for bindings must be unique on a
+case-insensitive basis, as well as field names of the source collection. If any names are not unique
+on a case-insensitive basis (ex: `myField` vs. `MyField`) the materialization will fail to apply.
+
+If necessary, you can add [projections](../../../concepts/advanced/projections.md) to your
+collection specification to change field names.
+
+## Delta updates
+
+This connector supports both standard (merge) and [delta updates](../../../concepts/materialization.md#delta-updates).
+The default is to use standard updates.
```

Comment:
Should we include a note to this here?

> 	// If there are multiple materializations operating on different tables within the same database
	// using the same metadata table path, they will need to concurrently update the checkpoints
	// table. Aquiring a table-level lock prevents "serializable isolation violation" errors in this
	// case (they still happen even though the queries update different _rows_ in the same table),
	// although it means each materialization will have to take turns updating the checkpoints
	// table. For best performance, a single materialization per database should be used, or
	// separate materializations within the same database should use a different schema for their
	// metadata.

Reply:
I would also suggest including a note regarding maximum length of strings / text fields and how we work it